content/release-9/_index.md

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
Our intention for https://github.com/jakartaee/jakarta.ee/issues/920 is to create the Jakarta 9 release page under content/release and migrate the content for the Jakarta 8 landing page under content/release/8.

Reply:
this is just a single component for randomization and committers preview on release page (not a full page template)

Reply:
You can check how it looks via [netlify link](https://deploy-preview-923--jakartaee.netlify.app/release-9/)

Reply:
For randomization [lodash sampleSize](https://lodash.com/docs/4.17.15#sampleSize) is used.

Reply:
+1 Just wanted to share our plans to make sure we don't overlap! This is great BTW!